index.css

```diff
@@ -0,0 +1,84 @@
+body {
+  box-sizing: border-box;
+  margin: auto;
+  background: white;
+  padding: 0;
+  font-family: Poppins, sans-serif;
+}
+
+.header-section {
+  height: 100vh;
+}
+
+.header-nav {
+  height: 7vh;
+  color: #6070ff;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.header-nav > a {
+  margin-left: 30px;
+  font-weight: bold;
+  text-decoration: none;
+  color: #6070ff;
+  text-decoration-color: none;
+  font-size: 22px;
+}
+
+.bar-line {
+  width: 30px;
+  height: 3px;
+  background: #6070ff;
+  margin: 4px;
+}
+
+.bar {
+  margin-right: 30px;
+}
+
+.header-herosection {
+  color: #172b4d;
+  background-image: url("/images/header-background.svg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 88vh;
+  width: 100vw;
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+}
+
+.header-herosection > h1 {
+  padding-top: 100px;
+  font-weight: bold;
+  font-size: 45px;
+  padding-left: 15px;
+  padding-right: 15px;
+}
+
+.header-herosection > p {
+  color: #344563;
+  padding: 0 20px 20px 20px;
+  font-size: large;
+}
+
+.header-herosection > h3 {
+  color: #7f8cff;
+  padding-left: 20px;
+}
+
+.social-media {
+  display: flex;
+  padding-left: 20px;
+}
```

Comment:
- **[OPTIONAL]**  Add the space at the end and the border radion on the left to be similar to the Figma template 👇🏽 

| Figma template | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/235256123-2e99fe5c-c3a9-4aec-bdb8-8d6768b281f1.png) | ![image](https://user-images.githubusercontent.com/58642949/235256199-455ca273-7437-469c-8a84-c04592aabe49.png) |